Jenkinsfile

```diff
@@ -338,6 +339,46 @@ pipeline {
         }
     }
 
+    stage('L2: NER with uncased Megatron') {
```

Comment:
These last two tests should be done in parallel (e.g. see parallel {} syntax used in ASR and SQUAD test). Just don't forget to put one on trainer.gpus=[0] and another one on trainer.gpus=[1]. Our jenkins workers are guaranteed to have 2 GPUs available.

Reply:
this results in an error in Jenkins, we'll add another nlp test there, just not 2 megatrons at once